### DISCUSSION TITLE
Does sexbound defeat work with SBR?

### DISCUSSION THREAD

**Discussion by Remis220** [ErinaSugino/Starbound-SBR-Mod#35] [General]
I installed defeat mod, but my character and NPC just die without transforming into sexnode. Can this be fixed?
Sorry for poor translation, i use translator.

**Comment by ErinaSugino** [comment]
Sexbound Defeat works about as well with SBR as it did with original SxB - for some people it miraculously works fine, for others not at all. That's to how error-prone Defeat itself always has been.

Plans to revive, fix and integrate Defeat into SBR exist for later down the roadmap, though no plans for now.
